### PR TITLE
Change made to enable opm new flags --skip-tls-verify and --use-http as --skip-tls flag is deprecated.

### PR DESCRIPTION
**Pull request by laxmikantbpandhare**
**Description of the change:**
User is not able to run the `operator-sdk run bundle` in local kind-registry as `--skip-tls` flag is deprecated. OPM introduced new flags `--skip-tls-verify and --use-http`. This fix will enable the newly introduced flags.

**Motivation for the change:**
Users will be able to execute `run bundle` in local registries without requiring any certifications.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [x] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Comment by laxmikantbpandhare** [comment]
will add test cases for the `--use-http` flag as well.

File: internal/olm/operator/registry/index/registry_pod.go [hunk 1]
```diff
@@ -77,8 +77,11 @@ type RegistryPod struct { //nolint:maligned
 	// The secret's key for this file must be "cert.pem".
 	CASecretName string
 
-	// SkipTLS controls wether to ignore SSL errors while pulling bundle image from registry server.
-	SkipTLS bool `json:"SkipTLS"`
+	// skipTLSVerify controls wether to ignore SSL errors while pulling bundle image from registry server.
+	SkipTLSVerify bool `json:"SkipTLSVerify"`
+
+	// skipTLSVerify controls wether to ignore SSL errors while pulling bundle image from registry server.
```
Comment (joelanford) [inline internal/olm/operator/registry/index/registry_pod.go:83]: Copy/paste typo here

**Review by joelanford — APPROVED** [review]
Looks great!
/approve

File: internal/olm/operator/registry/index/registry_pod.go (hunk 1 above)
Comment (laxmikantbpandhare) [reply internal/olm/operator/registry/index/registry_pod.go]: Yes, updated the description for UseHTTP variable.

**Review by rashmigottipati — APPROVED** [review]
/lgtm

File: internal/olm/operator/registry/index/registry_pod.go [hunk 2]
```diff
@@ -77,8 +77,11 @@ type RegistryPod struct { //nolint:maligned
 	// The secret's key for this file must be "cert.pem".
 	CASecretName string
 
-	// SkipTLS controls wether to ignore SSL errors while pulling bundle image from registry server.
-	SkipTLS bool `json:"SkipTLS"`
+	// SkipTLSVerify skip TLS certificate verification for container image registries while pulling bundles.
```
Comment (rashmigottipati) [inline internal/olm/operator/registry/index/registry_pod.go:80]:
```suggestion
	// SkipTLSVerify represents skip TLS certificate verification for container image registries while pulling bundles.
```

File: internal/olm/operator/registry/index/registry_pod_test.go [hunk 3]
```diff
@@ -94,23 +94,71 @@ var _ = Describe("RegistryPod", func() {
...
+
+			It("should return a container command with --ca-file", func() {
+				rp.CASecretName = caSecretName
+				output, err := rp.getContainerCmd()
+				Expect(err).To(BeNil())
+				Expect(output).Should(Equal(containerCommandFor(defaultDBPath, defaultBundleItems, true, rp.SkipTLSVerify, false)))
+			})
+
+			It("should return a container command for image with --skip-tls-verify", func() {
+				bundles := []BundleItem{defaultBundleItems[0]}
```
Comment (rashmigottipati) [inline internal/olm/operator/registry/index/registry_pod_test.go:108]: Check `len(defaultBundleItems) > 0` to ensure it has one item atleast before accessing defaultBundleItems[0].

**Review by varshaprasad96 — APPROVED** [review]
/lgtm
after addressing other reviews

File: internal/olm/operator/registry/index/registry_pod.go (hunk 2 above)
Comment (laxmikantbpandhare) [reply internal/olm/operator/registry/index/registry_pod.go]: Updated accordingly.

File: internal/olm/operator/registry/index/registry_pod_test.go (hunk 3 above)
Comment (laxmikantbpandhare) [reply internal/olm/operator/registry/index/registry_pod_test.go]: Yes, updated.

**Review by rashmigottipati — APPROVED** [review]

**Pull request merged**